scipy/stats/tests/test_continuous_basic.py

```diff
@@ -62,7 +62,7 @@
                    'erlang', 'f', 'gamma', 'gausshyper', 'gengamma',
                    'gennorm', 'genpareto', 'halfgennorm', 'invgamma',
                    'ksone', 'kstwobign', 'levy_l', 'loggamma', 'logistic',
-                   'maxwell', 'nakagami', 'ncf', 'nct', 'ncx2',
+                   'maxwell', 'nakagami', 'ncf', 'nct', 'ncx2', 'norminvgauss',
```

Comment:
This is because of `k1`/`k1e` do not handle complex-valued arguments, correct? In principle, `kv(1, z)`  and `kve(1, z)` work in the complex plane -- not that I require this change, I'm just saying it could be done.

Reply:
Correct